cromwell_tools/cromwell_tools.py

```diff
@@ -327,3 +334,61 @@ def generate_auth_header_from_key_file(json_credentials):
     scopes = ['https://www.googleapis.com/auth/userinfo.profile', 'https://www.googleapis.com/auth/userinfo.email']
     credentials = ServiceAccountCredentials.from_json_keyfile_name(json_credentials, scopes=scopes)
     return {"Authorization": "bearer " + credentials.get_access_token().access_token}
+
+
+def _localize_file(url, target_directory='.'):
+    """Localize file url to a directory. Supports local files and http endpoints
+
+    :param str url: url of local or http target
+    :param str target_directory: directory to localize file to
+    :return:
+    """
+
+    if not os.path.isdir(target_directory):
```

Comment:
`pathlib` is much easier for manipulating with the files and dirs here. Another good reason for supporting only `>=py3.5` 😄